htdocs/hrm/position_list.php

```diff
@@ -602,13 +605,30 @@
 	// Store properties in $object
 	$object->setVarsFromFetchObj($obj);
 
+
 	if ($mode == 'kanban') {
 		if ($i == 0) {
-			print '<tr><td colspan="'.$savnbfield.'">';
+			print '<tr><td colspan="12">';
 			print '<div class="box-flex-container">';
 		}
-		// Output Kanban
-		print $object->getKanbanView('');
+		// get info needed
+
+		$object->date_start = $obj->date_start;
+		$object->date_end = $obj->date_end;
+		$job = new job($db);
+		$job->fetch($obj->fk_job);
```

Comment:
We are inside a loop in a lis (that may be very long). In suche situation , we must never introduce call to a method that make sql subrequests.

If you have 500 lines in screen, it means you will make 501 SQL request instead of 1 increasing time to show the page by a factor  500 !
You must include all properties you need inside the main sql of the list so you have them already known in the result of the fetch. All you have to do is to make the set  userstatic->xxx = $obj->xxx

Reply:
ok i see i will fix it